rosetta/text/text_processors.py

```diff
@@ -685,6 +687,12 @@ def filter_sfile(
                 (2) idf(t, D) = log (N / M), where N is the total number of
                     documents in D and M is the number of documents in D which
                     contain the token t. The logarithm is base e.
+        filters : iterable over functions
```

Comment:
Can you add

* A comment that `filters` are implemented after `doc_id_filter` and before the tf idf filter.
* Tell the user to see the doc in `streaming_filters.py` for an explanation of what a record dict is (they don't necessarily know since it is our internal format) and examples of how to make filters.